addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -6348,16 +6351,20 @@ public Action:OnStomp(attacker, victim, &Float:damageMultiplier, &Float:damageBo
 		}
 		damageMultiplier=900.0;
 		JumpPower=0.0;
-		PrintHintText(victim, "Ouch!  Watch your head!");
-		PrintHintText(attacker, "You just goomba stomped somebody!");
+		PrintHintText(victim, "%t", "Player Goomba Stomped");
```

Comment:
Let's go a bit further and do something like `Player Goomba Stomped Boss` and `Boss Goomba Stomped Player`